style.css

```diff
@@ -0,0 +1,97 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: sans-serif;
+}
+body {
+  display: flex;
+
+  font-size: 30px;
+  height: 100vh;
+  justify-content: center;
+  align-items: center;
+  background: #fbfbfb;
+  color: #cacaca;
+}
+h1 {
+  font-size:20px;
+}
+.main-wrapper {
+  position: relative;
+  width: 260px;
+  height: 300px;
+  padding: 50px;
+  border-radius: 15px;
+  box-shadow: 2px 2px 20px rgba(0, 0, 0, 0.82);
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+.button-wrapper {
+  width: 160px;
+  height: auto;
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  overflow: hidden;
+}
+input {
+  display: none;
+}
+.entry {
+  position: relative;
+  width: 100%;
+  transition: 0.5s;
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-start;
+  align-items: center;
+  margin: 10px;
+}
+.circle {
+  border: 3px solid #cacaca;
+  border-radius: 50%;
+  cursor: pointer;
+  height: 32px;
+  width: 32px;
+  top: 50%;
+  transition: border-color 0.5s;
+}
+.entry-label {
+  cursor: pointer;
+  user-select: none;
+  -moz-user-select: none;
+  font-weight: 500;
+  padding: 0 0 0 15px;
```

Comment:
do you think `padding-left` is more readable ?

Reply:
i used shorthand prop

Reply:
surely i have mentined ```padding-left``` as per your suggestion